app/models/household_good.rb

```diff
@@ -1,9 +1,11 @@
 class HouseholdGood < ApplicationRecord
-  validates :weight, presence: true
-  validates :name, presence: true, uniqueness: { scope: :household_good_category_id }
-
   belongs_to :household_good_category
+
   validates :household_good_category, presence: true
+  validates :name, presence: true, uniqueness: { scope: :household_good_category_id }
+  validates :weight, presence: true
```

Comment:
There's no real required order for things in a model, but I've kinda tried to do:

```
- mixins
- miscellaneous stuff (e.g. `friendly_id` in some of the models)
- associations (e.g. `belongs_to`)
- validations
- scopes
- callbacks
- instance methods
- class methods
- private methods
```

So the change here is reordering based on a totally-not-at-all-official convention.